src/core/nixl_enum_strings.cpp

```
@@ -0,0 +1,85 @@
+    static const std::array strings = {
+        "DRAM_SEG", "VRAM_SEG", "BLK_SEG", "OBJ_SEG", "FILE_SEG"
+    };
+
+    if ((mem < 0) || (mem >= strings.size())) {
+        return "BAD_SEG";
+    }
+    return strings[mem];
+}
+
+std::string
+xferOpStr(const nixl_xfer_op_t &op) {
```

Cppcheck (2.20.0)

[style] The function 'xferOpStr' is never used.

(unusedFunction)